### ISSUE TITLE
[Feature Request] Include parameter normalize and scale X-axis when `predictions` outside [0, 1]

### ISSUE THREAD

**Issue by joaoOarthur** [Creditas/galeritas#21]
### Context

Quando passamos uma coluna `predictions` com previsoes fora do eixo [0, 1] precisamos normalizar os dados para esse intervalo e também arrumar o eixo X da `calibration_curve`

### Describe your proposed solution

- utilizar o parametro normalize da `sklearn.calibration_curve` utilizada no plot
- corrigir o eixo X para se adequar aos dados normalizados
